platform/ios/src/MGLMapView.mm

```diff
@@ -1866,6 +1890,10 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
             }
         }
     }
+    else if ([keyPath isEqualToString:@"bounds"] && [object isKindOfClass:NSClassFromString(@"_UILayoutGuide")])
```

Comment:
Comparing against the private class _UILayoutGuide is not particularly future proof.